src/host/ft_host/CJK_DbcsTests.cpp

```diff
@@ -278,8 +278,6 @@ class DbcsTests
     END_TEST_METHOD()
 };
 
-HANDLE hScreen = INVALID_HANDLE_VALUE;

```

Comment:
This turned out to be completely unused, so I deleted it altogether.

Reply:
Shouldn't we get a warning for this?  Also - what about warnings as errors?  Should this be turned on?  I am not familiar with the test projects or the settings.  (or what the current practice is for these types of things)   I was assuming all of the projects would have warnings as errors on but looking at the CI it doesn't appear so.  Is this a goal?  Should an issue be logged to make sure this is on for the projects?

Reply:
I'm not actually sure if Microsoft's compiler will warn for unused variables declared at non-static file scope - I couldn't find such a warning after a quick search, and I imagine it might not even be a good idea to have this be a warning, since another file may have the same variable declared extern.

One of the MSBuild files intended to be included by .vcxproj files - [src/common.build.pre.props](https://github.com/microsoft/terminal/blob/v0.1.1502.0/src/common.build.pre.props#L59-L66) - changes all enabled warnings to errors, but at the same time I have seen at least one compiler warning when running a clean build, so this script is probably not being included everywhere or is being overridden.

If you do create an issue to clean all existing compiler warnings out of the repo and enable warnings as errors everywhere, I'd definitely +1 it.

Reply:
@metathinker I didn't realize this was not in the function.  The diff threw me off.   I will put up an issue for the builds and warnings.   I want to take a look at the projects closer first.    [This](https://dev.azure.com/ms/Terminal/_build/results?buildId=18523) shows ten warnings.   Mostly around WinRt stuff.  I am not sure they are meaningful or not.  Assuming not.   Also, one C4267 which should probably be fixed.

Reply:
The WinRT stuff is already logged as 776.   I don't see any issue with C4267 warning.  I might have missed it.  I will look closer.